examples/native-hybrid-app/flutter_module/lib/main.dart

```diff
@@ -2,19 +2,21 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2022-Present Datadog, Inc.
 
+import 'package:datadog_flutter_plugin/datadog_flutter_plugin.dart';
+import 'package:datadog_tracking_http_client/datadog_tracking_http_client.dart';
 import 'package:flutter/material.dart';
 
 import 'my_app.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  // final config = DdSdkExistingConfiguration(
-  //   loggingConfiguration: LoggingConfiguration(),
-  //   detectLongTasks: true,
-  // )..enableHttpTracking();
+  final config = DatadogAttachConfiguration(
+    detectLongTasks: true,
+    reportFlutterPerformance: true,
```

Comment:
it is quite a difference with what was before, did we have it enabled before?

Reply:
In the example, no I don't think we did, but probably better to have it enabled.